x-pack/plugins/security_solution/server/endpoint/routes/resolver/tree/queries/base.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { JsonValue } from '@kbn/utility-types';
```

Comment:
Should we add/modify tests for this route to accommodate new timerange values.?

Reply:
There are no unit tests for any of this code, probably should be, but there are cypress and integration tests for these routes.